<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigSectionGeneral</class>
 <widget class="QWidget" name="ConfigSectionGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>278</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="labelSubTitle2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sample management</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_23">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Sample import</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboBufferSize">
     <item>
      <property name="text">
       <string>Default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">64</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">128</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">512</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">1024</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">2048</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">4096</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="labelSubTitle1">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Input / Output</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QCheckBox" name="checkBoucle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>When importing a sample, data located after the end of loop are automatically removed.
(same effect as the tool “Trim to end of loop”)</string>
     </property>
     <property name="text">
      <string>trim to loop</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QCheckBox" name="checkBlanc">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>When importing a sample, any blank area located before the beginning of the attack is automatically removed.
(same effect as the tool “Remove blank at start”)</string>
     </property>
     <property name="text">
      <string>remove blank at start</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboAudioOuput">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Audio backend</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>Buffer size</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>MIDI input</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="comboMidiInput"/>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_24">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Stereo editing</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="checkRepercussionStereo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>change linked sample</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboAudioOuput</tabstop>
  <tabstop>comboBufferSize</tabstop>
  <tabstop>comboMidiInput</tabstop>
  <tabstop>checkRepercussionStereo</tabstop>
  <tabstop>checkBoucle</tabstop>
  <tabstop>checkBlanc</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
